search.json

```diff
@@ -35,4 +35,14 @@ layout:
 "date":"{{ post.date }}"
 } {% unless forloop.last %},{% endunless %}
 {% endfor %}
-]
+{% for jobs in site.jobs %}
+{
```

Comment:
You should also create a condition that let's users know if the job is past the deadline, perhaps by prepending some text to the title if it is closed.